Docker_Container/README.md

```diff
@@ -0,0 +1,13 @@
+## Just put the whole project inside this folder
```

Comment:
There is no reference to this file from the root README.md. No one will know that this option exists if they read the installation documentation.